crates/turborepo-repository/Cargo.toml

```diff
@@ -26,6 +27,7 @@ turbopath = { workspace = true }
 turborepo-graph-utils = { path = "../turborepo-graph-utils" }
 turborepo-lockfiles = { workspace = true }
 turborepo-scm = { workspace = true }
+turborepo-telemetry = { version = "0.1.0", path = "../turborepo-telemetry" }
```

Comment:
This dependency makes me nervous. We have consumers of `turbo-repository` from outside of the Turborepo CLI

Reply:
Point taken, I will thread the error through and make sure to keep telem in lib only

Reply:
We also use it in turborepo-repository, which is OK for now, but I feel like that could very easily get consumed by other sources also. I will open a ticket for both of these to limit imports just so we are absolutely sure

Reply:
https://linear.app/vercel/issue/TURBO-2390/ensure-turborepo-telemetry-does-not-bleed-out-of-turbo-proper